src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -58,6 +61,10 @@ private void Render()
             TerminalBuffer.Initialize();
             // TODO: Fix. First line does not appear at top. Leaving empty line for now
             TerminalBuffer.WriteNewLine(string.Empty);
+
+            // Top line indicates the number of finished projects.
+            finishedProjectsLine = TerminalBuffer.WriteNewLine($"{completedProjects} projects finished building.");
```

Comment:
What if this was displayed on the titlebar (where it says "MSBuild - Build in progress") so that it is always visible to the user?